avocado/core/plugin_interfaces.py

```diff
@@ -162,6 +162,38 @@ def post_tests(self, job):
         """Entry point for job running actions after the tests execution."""
 
 
+class PreTest(Plugin):
+    """Base plugin interface for adding tasks before a test run.
+
+    This interface helps to create `avocado.core.nrunner.Task` for running
+    additional actions inside spawner environment before Test.
+    """
+
+    @abc.abstractmethod
+    def pre_test_runnables(self, test_runnable):
+        """Entry point for creating runnables, which will be run before test.
+        :param test_runnable: Runnable of the Test itself.
+        :return: PreTest task runnable.
```

Comment:
The description of the return implies a single runnable, while the `rtype` describes a list.  Can you please clear up the confusion here?